packages/@okta/vuepress-site/docs/release-notes/2021-okta-identity-engine/index.md

```diff
@@ -4,6 +4,18 @@ title: Okta Identity Engine API Products Release Notes 2021
 <ApiLifecycle access="ie" /><br>
 <ApiLifecycle access="Limited GA" /><br>
 
+## October
+
+### Monthly Release 2021.10.0
+
+| Change                                                                   | Expected in Preview Orgs |
+|--------------------------------------------------------------------------|--------------------------|
+| [Brands API option to remove sign-in page footer message](#brands-api-option-to-remove-sign-in-page-footer-message) | October 6, 2021 |
+
+#### Brands API option to remove sign-in page footer message
```

Comment:
```suggestion
#### New Brands API option to remove sign-in page footer message
```

just feels like it needed the word "new" to clarify if it was new or being taken away or what....